pylintrc.tests

```diff
@@ -0,0 +1,80 @@
+[MASTER]
```

Comment:
We need a separate pylint config file for tests because we need different rules for tests. Unfortunately there's no way for a single pylintrc file to apply different rules to different files/directories. Nor can pylintrc files inherit from each other

Reply:
These two aren't following the pattern we established in [h-cookiecutter-pypackage](https://github.com/hypothesis/h-cookiecutter-pypackage/) which is to have a `.pylintrc` and `tests/.pylintrc`. I think the idea was:

 * It's the default filename, and so is unsurprising / we didn't have much imagination
 * Running pylint manually from either directory has a descent chance of picking up the right config

I think it would be nicer to have everything going in the same direction if possible.

The comments say that `prospector` will pick this up and run `pylint` if it's about, but I think we could avoid that by adding `--without-tool pylint`, or positively listing the tools we do want to run (which might be nice anyway).

Reply:
I've changed these to `.pylintrc` and `tests/.pylintrc` in [the other pr](https://github.com/hypothesis/lms/pull/1523)